client/src/types/userTypes.ts

```diff
@@ -43,9 +43,9 @@ export interface IUser {
   isAuth: boolean;
   hasError: boolean;
   id: number | null;
-  driver_info?: IDriver,
-  isDriver: boolean,
-  authError: string,
+  driver_info?: IDriver;
```

Comment:
Why you use snake case formatting?

Reply:
Resolved (will be changed in the next pull requests)